docs/guides/options.md

```diff
@@ -205,47 +210,24 @@ See [the plugins guide][plugins] for more information on Video.js plugins.
 
 ### `sourceOrder`
```

Comment:
Hm..., I'd have to double check but I think this option is removed completely due to middleware and now it's always per source.